File: QRCode.Core/Domain/product.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QRCode.Core.Domain
{
    public class product
    {
        public virtual long qrproductid { get; set; }
        public virtual string name { get; set; }
        public virtual string code { get; set; }
        public virtual string category { get; set; }
        public virtual string url_img { get; set; }
        public virtual string url_video { get; set; }
        public virtual string url_iso { get; set; }
        public virtual string url_barcode { get; set; }
        public virtual decimal price { get; set; }
        public virtual string slogan { get; set; }
        public virtual string logo { get; set; }
        public virtual string des_story { get; set; }
        public virtual string des_pack { get; set; }
        public virtual string des_element { get; set; }
        public virtual string des_uses { get; set; }
        public virtual string des_guide { get; set; }
        public virtual string des_preserve { get; set; }
        public virtual string des_startdate { get; set; }
        public virtual string des_enddate { get; set; }
        public virtual string additional { get; set; }
        public virtual DateTime? created_date { get; set; }
        public virtual decimal created_by { get; set; }
        public virtual DateTime? lastcreated_date { get; set; }
        public virtual decimal lastcreated_by { get; set; }
        public virtual bool status { get; set; }

    }
}

File: QRCode.FEW/ClientApp/src/app/models/product.ts
import { enterprisview } from "./qr_enterprise";

export interface product {
    qrproductid: number;
    name: string;
    code: string;
    category: string;
    price: number;
    slogan: string;
    logo: string;
    url_img: string;
    url_iso: string;
    url_barcode: string;
    des_story: string;
    des_manufactur: string;
    des_pack: string;
    des_element: string;
    des_uses: string;
    des_guide: string;
    des_preserve: string;
    des_startdate: string;
    des_enddate: string;
    url_video: string;
    lastcreated_date: Date;
    lastcreated_by: number;
    additional: string;
    status: boolean;
    created_by: number;
    created_date: Date;
    enterpriseid:number;
}
export interface productview {
    qrproductid: number;
    name: string;
    code: string;
    //category: string;
    price: number;
    //slogan: string;
    //logo: string;
    url_img: string;
    //url_iso: string;
    //url_barcode: string; 
    des_story: string;
    des_manufactur: string;
    des_pack: string;
    des_element: string;
    des_uses: string;
    des_guide: string;
    des_preserve: string;
    des_startdate: string;
    des_enddate: string;
    //url_video: string;  
    additional: string;
    enterpriseid: number;
    list_ref: productview[];
    congty: enterprisview;
    list_img:string[];

}
export interface user_reset {
    email: string;
    password_old: string;
    password: string;
}
export interface product_excel {
    name: string;
    code: string;
    category: string;
    price: string;
    slogan: string;
    logo: string;
    url_img: string;
    url_iso: string;
    url_barcode: string;
    des_story: string;
    des_manufactur: string;
    des_pack: string;
    des_element: string;
    des_uses: string;
    des_guide: string;
    des_preserve: string;
    des_startdate: string;
    des_enddate: string;
    url_video: string;
    additional: string;
    enterprise: string;
    err_str: string;
}